### DISCUSSION TITLE
bitwarden网站字体图标不显示解决办法

### DISCUSSION THREAD

**Discussion by weavers1f** [F9y4ng/GreasyFork-Scripts#401] [问答专区(Question / Answer)]
https://bitwarden.com/help/ 这网站上，那些字体图标都不显示，自己也没有网页编程经验，按照网站样式错误修正的设置分享那里写的类比尝试了下，发现是在需要渲染的网页元素：文本域 追加:not(span.css-hez0d6)可以解决
![image](https://github.com/user-attachments/assets/a70c081d-bb01-4d34-ad54-d39a2f6bb95f)

**Comment by F9y4ng** [comment]
下个月更新中会增加对`bwi-font`字体图标的支持。

**Reply by F9y4ng** [reply]
及时更新至 v2024.12.07.1 即可。
